infra/modules/aws-psoxy-rest/main.tf

```diff
@@ -63,23 +63,12 @@ Review the deployed function in AWS console:
 
 ### Prereqs
 Requests to AWS API need to be [signed](https://docs.aws.amazon.com/general/latest/gr/signing_aws_api_requests.html).
-One tool to do it easily is [awscurl](https://github.com/okigan/awscurl). Install it:
+Our Node.js based testing tool does it, but the machine running the testing script must have
+the appropriate AWS credentials (you can use [aws-mfa](https://github.com/broamski/aws-mfa) or any
+similar tool).
 
-On MacOS via Homebrew:
-```shell
-brew install awscurl
-```
-
-Alternatively, via `pip` (python package manager):
-```shell
-pip install awscurl
-# Installs in $HOME/.local/bin
-# Make it available in your path
-# Add the following line to ~/.bashrc
-export PATH="$HOME/.local/bin:$PATH"
-# Then reload the config
-source ~/.bashrc
-```
+- [Node.js] >= v16
+- [npm] >= v8
```

Comment:
don't they also need to install the tool (`npm i`) in the `tools/psoxy-test`? 

can we give them that command in the markdown as an absolute path? eg, `npm -i ${var.path_to_repo_root}tools/psoxy-test/`